contents/docs/debugging/index.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: Debugging Zero
```

Comment:
didn't realize you had already started adding a debug section!

Guess we can close https://github.com/rocicorp/zero-docs/pull/99